dist/test/benchmark.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
```

Comment:
why are the tests ending up in the `dist` directory?

Reply:
Whoops, didn't commit the .tsconfig `exclude` change

Reply:
I had excluded `"**/*.spec.ts"` although this won't exclude `test/benchmark.ts`, though. I'm not entirely sure where we should put Volumetric & Performance stuff. Thoughts?

Reply:
new directory called `benchmarks` or something?